mcs/tools/mkbundle/template_common.inc

```diff
@@ -18,8 +18,6 @@ void initialize_mono_api (const BundleMonoAPI *info)
 	mono_api.mono_register_machine_config = info->mono_register_machine_config;
 }
 
-#ifdef USE_COMPRESSED_ASSEMBLY
```

Comment:
Why did you undo this change?

Reply:
If you use mkbundle and disable USE_COMPRESSED_ASSEMBLY, it doesn't build currently. This change removed too much code. I intend to find the commit that made the problem and fix it correctly, this was a prototype to see if I could get it working well without investing much in it.

I intend to circle back and fix this for my next sprint, as I want to also benchmark the impact of this change and see if some knobs should be tuned. We probably want more than just direct-icalls with the normal AOT options if we're special casing mscorlib.